docs/attribution/0001-analytics.rst

```diff
@@ -359,17 +360,30 @@ For all links to accounts.firefox.com use these data attributes (* indicates a r
 +-----------------------+----------------------------------------------------------------------------------+
 | Data Attribute        | Expected Value                                                                   |
 +=======================+==================================================================================+
-| ``data-cta-type`` *   | fxa-servicename (e.g. ``fxa-sync``, ``fxa-monitor``)                             |
+| ``data-cta-text`` *   | Text or name of the link (e.g. ``Sign Up``, ``Join Now``, ``Start Here``).       |
 +-----------------------+----------------------------------------------------------------------------------+
-| ``data-cta-text``     | Name or text of the link (e.g. ``Sign Up``, ``Join Now``, ``Start Here``).       |
-|                       |                                                                                  |
-|                       | We use this when the link text is not useful, as is the case with many           |
-|                       | account forms that say, ``Continue``. We replace ``Continue`` with ``Register``. |
+| ``data-cta-type`` *   | fxa-servicename (e.g. ``fxa-sync``, ``fxa-monitor``)                             |
 +-----------------------+----------------------------------------------------------------------------------+
 | ``data-cta-position`` | Location of CTA on the page (e.g. ``primary``, ``secondary``, ``header``)        |
 +-----------------------+----------------------------------------------------------------------------------+
 
 
+Link Click
+~~~~~~~~~
+
+Our implementation of ``link_click`` for GA4 is based of the existence of certain data-attributes on a
+link element.
+
+`data-link-text` must be present to trigger the event:
+
+- data-link-text (examples: “Monitor”, “Features”, “Instagram (mozilla)”, “Mozilla VPN”)
+- data-link-position (examples: topnav, subnav, sticky-promo, topnav - firefox, footer - company)
```

Comment:
Is `data-link-type` still used in GA4 at all? If so, is it worth making clear when it needs to be used? (I'm assuming for some other event?)

Reply:
At the moment it is not used in GA4.